content/blog/animortis/_my_first_week_in_haiku.md

```diff
@@ -0,0 +1,39 @@
++++
+type = "blog"
+title = "My First Week in Haiku"
+author = "Animortis"
+date = "April 22, 2021"
+tags = ["haiku", "software"]
++++
+
+As you boot your first non-Windows, non-Unix system, you may have a moment when you wonder if this thing is going to work. It’s not from the same family of systems you’re used to: It’s Haiku, and it’s totally different and unique. The great part, though, is when it works better than you could’ve possibly imagined.
```

Comment:
Not a huge deal but my understanding is not to capitalize after a colon, so this could be "...used to: it's Haiku..."

Though I just read online and it is debatable so either way.

Similar with the one below "...Haiku: The experimental successor..."

Reply:
Hey, thanks for the notes! So I try to stick to news-publication style (AP Style) and the guideline there is to capitalize if the items after the colon form a complete sentence. You would be correct if the items were just a list or something, though (Example: "It was this: one, two, three."). I do appreciate the check! Clearly I missed some stuff this late.